frontend/src/app/app.component.ts

```diff
@@ -71,56 +74,58 @@ export class AppComponent {
           this.addResultToOutputLog(this.fetchedResult);
         },
         (error) => {
-          this.addErrorToOutputLog(error);
+          this.addErrorToOutputLog(`${this.priceDestination}`, error);
         }
       );
     }
   }
 
   /**
-   * Creates a "log" that will be displayed in the ui
-   *
-   * @param message that should be displayed
-   */
-  printRequestType(message) {
-    this.consoleOutput.push({
-      message: message,
-      type: 'info',
-    });
-  }
-
-  /**
-   * Sends a get request to the database service
+   * Sends a post request to the backend with the necessary request information which uses it
+   * to send a HTTP get request to the database service.
    *
    * @param url of the get request
+   * @return data of the request
    */
   getRequestDatabaseService(requestServiceUrl: string): any {
-    console.log('hi');
     return this.http.post(this.backendDestination, {
       requestService: requestServiceUrl,
       endpoint: this.requestName,
     });
   }
 
+  /**
+   * Appends a message containing the result of the request to the output log array.
+   * The content in the output log array are visualized in the ui.
```

Comment:
inhalt gut.
aber: entweder 'content [...] **is** visualized' oder content**s** [...] are visualized'.
plural und singular mischen ist einfach nicht so toll.